packages/api/src/command/medical/patient/__tests__/store-query-cmd.ts

```diff
@@ -39,14 +42,21 @@ export const documentQueryProgress: DocumentQueryProgress = {
   convert: makeProgress(),
 };
 
-export const consolidatedQuery: QueryProgress = {
-  status: "processing",
-  startedAt: new Date(),
-};
+export const consolidatedQueries: ConsolidatedQuery[] = [
+  {
+    requestId,
+    status: "processing",
+    startedAt: new Date(),
+    resources: ["Account"],
+    conversionType: "pdf",
+    dateFrom: new Date().toISOString(),
+    dateTo: new Date().toISOString(),
+  },
+];
```

Comment:
Could we add a few more queries here so the tests show how this works when there are multiple queries?

Reply:
and also where there are querres where both are of conversion type pdf but have different other params

Reply:
Create a function instead to create whatever permutation